ports/raspberrypi/boards/rfguru_rp2040/pico-sdk-configboard.h

```diff
@@ -0,0 +1 @@
+// Put board-specific pico-sdk definitions here. This file must exist.
```

Comment:
Not needed to accept, but you can consider adding this if you are seeing xtal osc startup problems on some samples:
https://github.com/adafruit/circuitpython/blob/a6c6d099fd8c3e7ccb70602837b38dffcd8c1535/ports/raspberrypi/boards/adafruit_feather_rp2040/pico-sdk-configboard.h#L3-L4